### ISSUE TITLE
Do not always package chart

### ISSUE THREAD

**Issue by orifito** [jfrog/chartcenter-plugin#5]
I'm migrating all our charts dependencies to use chartcenter, ended up having to use this plugin. However we only need to update dependencies, don't package the chart.

Or I can always wait for [this](https://github.com/helm/helm/pull/8875) to be merged...

**Comment by bhiravabhatla** [comment]
+1 @orifito , Please have an option to avoid packaging and just do update dependencies.

**Comment by rimusz** [comment]
@orifito @bhiravabhatla you do not need to use this plugin to update dependencies.
`helm dependency update` works just fine https://github.com/jfrog/chartcenter-plugin/blob/master/scripts/center.sh#L26
The `helm package` is the issue :(

**Comment by orifito** [comment]
@rimusz indeed, I've been using `helm dep up` so far without issues. Just found the `helm package` "issue" when migrating to chartcenter

**Comment by rimusz** [comment]
this is why `helm package` "issue" is sorted by this plugin :)

**Comment by orifito** [comment]
Yeah I can always forget about this plugin :D

**Comment by bhiravabhatla** [comment]
so, basically my scenario is -> 

Say, I have below subchart dependency - for eg, elasticsearch  (elastic official chart).  _stable_ is my alias to Jfrog chartcenter (which is accessed by a virtual repo in Jfrog artifactory).

```
dependencies:
  - name: elastic/elasticsearch
    repository: "@stable"
```
- Run dependency update
-  It downloads chart to $(CHART_NAME)/charts folder as tgz file - for example  $(CHART_NAME)/charts/elasticseach.tgz
- I run helm upgrade or helm install ->
I get this error 

```
helm.go:81: [debug] found in Chart.yaml, but missing in charts/ directory: stable/elasticsearch

```

**Comment by bhiravabhatla** [comment]
I am not sure above issue could be fixed by plugin. It looks more like a helm install/upgrade issue. I am able to download the dependencies. - not able run install/upgrade.

While installing the subchart - its trying match the exact name (namespaced name) with files in charts/ folder - which it would not find  :(

**Comment by rimusz** [comment]
yes, that's correct, helm cli doesn't understand `elastic` namespace in `elastic/elasticsearch`
